containers/proxy/config/conf.d/tracing.conf

```diff
@@ -1,8 +1,14 @@
 # this creates a tracing ID that will be propagated to all following requests.
 # it will use the current RequestID but prefer the value from an incoming X-Tracing-ID header.
 # this allows re-using the tracingID for sub requests made by an application.
-# example: user->proxy->webserver->php (Uvex TYPO3)->proxy->webserver->php (Uvex Shopware) will use the same tracingID
+# example: user->proxy->webserver->php->proxy->solr will use the same tracingID
 map $http_x_tracing_id $traceId {
-    default   $http_x_tracing_id;
-    ""        $request_id;
-}
\ No newline at end of file
+	# the incoming header is empty (or does not exist) -> use current requestID
+	"" $request_id;
+
+	# the incoming header is a hex string -> use it
+	"~^[0-9a-fA-F]+$" $http_x_tracing_id;
```

Comment:
We can think about also validating the length of the header. 
`$request_id: unique request identifier generated from 16 random bytes, in hexadecimal`
We could use  `~^[0-9a-fA-F]{1,16}$`. This give the developer the ability to choose a shorter one.
Otherwise you could spam the log with very long header values.

Reply:
Is 1 a reasonable minimum? How would one use a single digit tracing id? I would require a minimum of at least 4, better 8 digits

Reply:
Why have a lower limit at all? Let's set it to exactly 16 chars.